src/test/java/sg/dex/starfish/impl/TestResources.java

```diff
@@ -15,18 +17,12 @@
 public class TestResources {
 
     @Test
-    public void testResourceAssetWithContentHash() {
+    public void testResourceAssetWithContentHash() throws DigestException {
         DataAsset dataAsset = ResourceAsset.create("assets/hello.txt");
         byte[] bs = dataAsset.getContent();
-        String s = new String(bs, StandardCharsets.UTF_8);
-        Assertions.assertEquals("Hello Starfish", s);
```

Comment:
Why are we deleting these lines of tests?

Reply:
updated